Xamarin.Forms.Platform.Android/FastRenderers/FrameRenderer.cs

```diff
@@ -234,6 +235,14 @@ protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEv
 				UpdateCornerRadius();
 			else if (e.PropertyName == Frame.BorderColorProperty.PropertyName)
 				UpdateBorderColor();
+			else if (e.Is(Xamarin.Forms.Layout.IsClippedToBoundsProperty))
+				UpdateClippedToBounds();
+		}
+
+		void UpdateClippedToBounds()
+		{
+			if (!_disposed)
+				ClipToOutline = Element.IsClippedToBounds;
```

Comment:
Needs api check
https://developer.android.com/reference/android/view/View.html#setClipToOutline(boolean)

I added a helper for this with checkbox
 https://github.com/xamarin/Xamarin.Forms/blob/4.1.0/Xamarin.Forms.Platform.Android/ViewExtensions.cs#L81